sys-process/bashtop/bashtop-0.9.23.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="Resource monitor that shows usage and stats"
+HOMEPAGE="https://github.com/aristocratos/bashtop"
+SRC_URI="https://github.com/aristocratos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+DOCS=( README.md )
+
+DEPEND="${RDEPEND}"
+RDEPEND=""
```

Comment:
All of these could be removed from the ebuild making it cleaner, if you wish.

Reply:
What are your suggestions? @thesamesam @dol-sen